crates/bevy_pbr/src/meshlet/visibility_buffer_software_raster.wgsl

```diff
@@ -44,28 +44,30 @@ fn rasterize_cluster(
     let meshlet_id = meshlet_cluster_meshlet_ids[cluster_id];
     let meshlet = meshlets[meshlet_id];
 
-    // Load and project 1 vertex per thread
-    let vertex_id = local_invocation_index;
-    if vertex_id < meshlet.vertex_count {
-        let meshlet_vertex_id = meshlet_vertex_ids[meshlet.start_vertex_id + vertex_id];
-        let vertex = unpack_meshlet_vertex(meshlet_vertex_data[meshlet_vertex_id]);
-
-        // Project vertex to viewport space
-        let instance_id = meshlet_cluster_instance_ids[cluster_id];
-        let instance_uniform = meshlet_instance_uniforms[instance_id];
-        let world_from_local = affine3_to_square(instance_uniform.world_from_local);
-        let world_position = mesh_position_local_to_world(world_from_local, vec4(vertex.position, 1.0));
-        var clip_position = view.clip_from_world * vec4(world_position.xyz, 1.0);
-        var ndc_position = clip_position.xyz / clip_position.w;
+    let instance_id = meshlet_cluster_instance_ids[cluster_id];
+    let instance_uniform = meshlet_instance_uniforms[instance_id];
+    let world_from_local = affine3_to_square(instance_uniform.world_from_local);
+
+    // Load and project 1 vertex per thread, and then again if there are more than 128 vertices in the meshlet
+    for (var i = 0u; i <= 128u; i += 128u) {
```

Comment:
nit: up to you but it feels like `i < 255u` would be more clear; in general, this loop is

for (i = 0; i < max_vertices; i += workgroup_size)

Reply:
Ah, not quite the way I thought of it. My thought process was "we want to do this twice, with the second time using an offset of 128".